### PR TITLE
Only show nodes found by the query in the pivot table. Resolves #43

### LINKED ISSUE #43: Remove analysis of nodes' properties by sentence position
This is superseded by the ability to analyse specifically matched nodes #3.

### PR DESCRIPTION
**Pull request by oktaal**

### CODE REVIEW

File: ts/analysis-component.ts [hunk 1]
```diff
@@ -59,6 +59,7 @@ export class AnalysisComponent {
         let renderers = $.extend($.pivotUtilities.renderers,
             { 'File export': (new FileExportRenderer()).render });
         let pivotData = this.analysisService.getFlatTable(searchResults, this.variables.map(x => x.name), metadataKeys);
+        let defaultVariable = this.variables.length > 0 ? [this.variables[0].name.substr(1)] : [];
         return element.pivotUI(
```
Comment (BeritJanssen) [inline ts/analysis-component.ts]: Why do you make an array from defaultVariable, if - as far as I can see - you only use one value?

File: ts/analysis-component.ts [hunk 2]
```diff
@@ -69,8 +70,8 @@ export class AnalysisComponent {
                     'First': utils.aggregators['First'],
                     'Last': utils.aggregators['Last']
                 },
-                rows: ['lem1'],
-                cols: ['pos1'],
+                rows: defaultVariable.map(v => `lem_${v}`),
+                cols: defaultVariable.map(v => `pos_${v}`),
                 renderer: heatmap,
```
Comment (BeritJanssen) [inline ts/analysis-component.ts]: What happens if defaultVariable = []? Are rows and cols then undefined? If so, is this the intended behaviour?

Comment (oktaal) [reply ts/analysis-component.ts]: Yes, the table will be empty then. The purpose of this is to show the currently selected rows and columns of the pivot table. The user can always adjust add/remove (meta) variables from the interface.

File: ts/analysis-component.ts (hunk 1 above)
Comment (BeritJanssen) [reply ts/analysis-component.ts]: Thanks for the clarification!

**Pull request merged**